frontend/src/aboutme.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './about.css';
+
+export default function About(){
+  return (
+    <main>
+      <div class="about">
+        <h2>About Me</h2>
+        <p>
+          I am a first year computer science major, that knows absolutely nothing about cooking. So take all these recipes with a grain of salt. I don't have much else to say so here is a picture of me. :)
+          </p>
+
+        </div>
```

Comment:
This all looks good and functions fine, only thing to make sure of is all your tags are indented properly. Tedious, but when we use linting in project teams this will crash your site.